README.md

```diff
@@ -1,5 +1,15 @@
 # downloadTwitterPictures
 
+## Fork from miguelmalvarez
```

Comment:
Can you please remove the "fork from miguel"? This way, I will be able to merge directly